packages/twenty-front/src/modules/object-record/record-table/record-table-footer/hooks/useAggregateRecordsForRecordTableColumnFooter.tsx

```diff
@@ -32,13 +32,10 @@ export const useAggregateRecordsForRecordTableColumnFooter = (
     recordIndexViewFilterGroups,
   );
 
-  const viewFieldId = currentViewWithSavedFiltersAndSorts?.viewFields?.find(
-    (viewField) => viewField.fieldMetadataId === fieldMetadataId,
-  )?.id;
-
-  if (!viewFieldId) {
-    throw new Error('ViewField not found');
-  }
+  const viewFieldId =
```

Comment:
Just curious, is it not dangerous to not throw anymore and fallback to ''? I feel like it might hide an underlying issue? What motivated you to do this change?

Reply:
I saw this with @lucasbordeau. If the id is "" it will fail later. The idea is not to have a hook that throws in the declaration part, but to save the errors for the "actions" (here, the moment we actually fetch data). It does still feel quite hacky though, I agree!

Reply:
Got it, thanks @ijreilly